hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -1072,6 +1072,21 @@ public AclStatus getAclStatus(final Path path) throws IOException {
     }
   }
 
+  public void access(final Path path, final FsAction mode)
```

Comment:
Could you update this method to include "AbfsPerfInfo" introduced by HADOOP-16612?